### PR TITLE
Niamh25/doc 5218/getting started gradle plugin images update

### PR DESCRIPTION
**Pull request by Niamh25**
Updated three images in the CSDE and configuring the Network participants. Added new image to the configuring the CSDE section.

### CODE REVIEW

**Comment by Niamh25** [comment]
Preview found here: 
https://niamh.preview.docs.r3.com/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.html

File: content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md [hunk 1]
```diff
@@ -64,14 +64,16 @@ The CSDE includes [Gradle tasks](#gradle-helpers-for-the-combined-worker) to man
 
 {{< figure src="gradle-configuration.png" figcaption="Gradle Java version" alt="JVM version in IntelliJ for CSDE project" >}}
 
+{{< figure src="configure-CSDE-build-gradle-block.png" figcaption="CSDE build gradle" alt="CSDE build gradle block in IntelliJ for CSDE project" >}}
```
Comment (tlawson3) [inline content/en/platform/corda/5.0-beta/developing/getting-started/cordapp-standard-development-environment/csde.md:67]: Can we change the caption to 

> Configure the CSDE plugin via the build.gradle file

**Review by tlawson3 — CHANGES_REQUESTED** [review]

**Review by yarivus — COMMENTED** [review]
I see pending comments from @tlawson3

**Review by tlawson3 — APPROVED** [review]
💯

**Review by AnnaK-R3 — DISMISSED** [review]

**Review by amiegrace12 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @Niamh25

**Pull request merged**